apps/condo/domains/resident/schema/FindOrganizationsForAddressService.js

```diff
@@ -0,0 +1,122 @@
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+const checkLimits = async (uniqueField) => {
+    await redisGuard.checkCustomLimitCounters(
+        `find-organizations-by-address-${uniqueField}`,
+        RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC,
+        MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC,
+    )
+}
+
+const FindOrganizationsForAddressService = new GQLCustomSchema('FindOrganizationsForAddressService', {
+    types: [
+        {
+            access: true,
+            type: 'input FindOrganizationsForAddressInput { addressKey: String!, unitName: String, unitType: String, tin: String, accountNumber: String }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressOrganizationType { id: ID!, name: String!, tin: String!, type: OrganizationTypeType! }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressAccountType { number: String!, category: ID!, balance: String, routingNumber: String!, bankAccountNumber: String! }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressOutput { organization: FindOrganizationForAddressOrganizationType!, account: FindOrganizationForAddressAccountType, hasMeters: Boolean!, hasBillingData: Boolean! }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canFindOrganizationsForAddress,
+            schema: 'findOrganizationsForAddress (data: FindOrganizationsForAddressInput!): [FindOrganizationForAddressOutput]',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { addressKey, unitName, unitType, tin, accountNumber } = data
+                if (context.authedItem.type === RESIDENT) {
+                    await checkLimits(context.authedItem.id)
+                }
+                const properties = await find('Property', { addressKey, deletedAt: null })
+                if (!properties) {
+                    return []
+                }
+                let organizations = await find('Organization', {
+                    id_in: [...new Set(properties.map(({ organization }) => organization))],
+                    ...tin ? { tin } : {},
+                    deletedAt: null,
+                })
+                const organizationIds = organizations.map(({ id }) => id)
+                const withAcquiring = await getOrganizationsWithAcquiring(organizationIds)
+                organizations = organizations.filter(({ id }) => withAcquiring.has(id))
+                const withMeters = await getOrganizationsWithMeters(organizationIds)
+                const billingInformation = await getBillingInformationForOrganizations(organizations)
+                organizations = organizations.filter(organization => organization.hasMeters || billingInformation[organization.id])
```

Comment:
Need clarification about this part

Reply:
You will receive a list of organizations that have some agreements with us (acquiring context in finished status) and has receipts or meters